.circleci/config.yml

```diff
@@ -144,6 +144,10 @@ commands:
       - restore_cache:
           keys:
             - det-react-deps-v1dev3-{{ checksum  "webui/react/package-lock.json" }}
+      - attach_workspace:
+          at: .
+      - run: make -C webui/api-ts-sdk get-deps-package
+      - run: make -C webui/api-ts-sdk build-package
```

Comment:
blocking: let's move lines 147-150 to be before `restore_cache`. This way we don't break up the "restore cache, get dependencies, save cache" workflow.